core/parentchain/block-importer/src/block_importer.rs

```diff
@@ -163,9 +163,7 @@ impl<
 			// Execute indirect calls that were found in the extrinsics of the block,
 			// incl. shielding and unshielding.
 			match self.indirect_calls_executor.execute_indirect_calls_in_extrinsics(&block) {
-				Ok((unshielding_call_confirmations, executed_shielding_calls)) => {
-					// Include all unshielding confirmations that need to be executed on the parentchain.
-					calls.extend(unshielding_call_confirmations.into_iter());
```

Comment:
No, we did not "use" it, in a sense of code using it. However, the case of notifying the client about an executed indirect call, is not yet solved.. and linked to this I suppose. @clangenb  What do you think?

Reply:
Yes, we wanted to keep those such that a client can query the state of executed indirect calls. Having the information in the sidechain block is enough - if we have a sidechain. If we don't have a sidechain, we need to keep them, even if we don't expose an API yet to query that: #765

Reply:
So that means these changes are acceptable for now? 
We have to be aware that if we have no sidechain, we don't have the ability to execute indirect calls anymore now, since the execution of operations from the top pool is linked with the sidechain block production.

Reply:
Ok, let's accept this for know. 👍